OrbitCore/LinuxUtils.cpp

```diff
@@ -121,6 +122,24 @@ ErrorMessageOr<std::vector<ModuleInfo>> ListModules(int32_t pid) {
   return result;
 }
 
+ErrorMessageOr<std::vector<TracepointInfo>> ListTracepoints() {
```

Comment:
Add a smoke test for this method, you could check that it returns some of tracepoints every linux kernel expected to have.

Reply:
A unit test or a simple log would be enough?